code/game/objects/items/weapons/staples.dm

```diff
@@ -91,7 +91,10 @@
 		var/armor = H.run_armor_check(O, "melee")
 		if(armor <= 40)
 			H.throw_alert("embeddedobject", /obj/screen/alert/embeddedobject)
-			var/obj/item/organ/limb/L = pick(H.organs)
+			var/obj/item/organ/limb/L = H.getrandomorgan()
```

Comment:
Yeah, fair enough. It's a leftover proc from before I rewrote limbs not to create dummy ones on dismember.